gulpfile.js

```diff
@@ -90,10 +95,36 @@ gulp.task('lint:sass', function () {
             config: './scss-lint.json'
         }));
 });
+*/
 
 // #############################################################################
 // PREPROCESSING
-gulp.task('preprocess', ['images', 'docs']);
+gulp.task('preprocess', ['sass', 'images', 'docs']);
+
+gulp.task('sass', function () {
+    gulp.src(PROJECT_PATTERNS.sass)
+        // sourcemaps can be activated through `gulp sass --debug´
+        .pipe(gulpif(DEBUG, sourcemaps.init()))
+        .pipe(sass())
```

Comment:
Is it possible to always compress the outputted css ?
{ outputStyle: 'compressed' }